src/Aspire.Hosting.AWS/CloudFormation/CloudFormationAWSConsoleUrlMapper.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.AWS.CloudFormation;
+
+internal static class CloudFormationAWSConsoleUrlMapper
+{
+    public static string MapCloudFormationUrl(string stackId)
+    {
+        var region = stackId.Split(":")[3];
+
+        return
+            $"https://{region}.console.aws.amazon.com/cloudformation/home?region={region}#/stacks/resources?stackId={stackId}";
```

Comment:
There are some corner cases for knowing the console host. For example the China regions go to separate host. It is also possible users could configure the service client being used here to point to Localstack in which case you wouldn't have a console to go to.

I suggest passing in the `IAmazonCloudFormation` service client into this method. Then call the following method to determine the endpoint the service client is going to send the request to.
```
var cloudFormationUrl = client.DetermineServiceOperationEndpoint(new DescribeStacksRequest { StackName = stackId }).URL;
```

Then in the host of that URL look for `.amazonaws.`. If you don't find that token then don't add a url to the dashboard. If you do find one then take from the service url from the `amazonaws.` to the end to create the suffix of the console endpoint.

Reply:
Using `DetermineServiceOperationEndpoint` I received the following URL: `https://cloudformation.eu-west-1.amazonaws.com/`. Filtering for `.amazonaws.` as suggested is not a problem. However, I'm a unsure what part of this can be used to build the full console URL. 

The URL should be `https://console.aws.amazon.com/cloudformation/home?region={region}#/stacks/resources?stackId={stackId}` right?

If I'm passing the client that has just been used successfully, then I should be able to use region from `client.Config.RegionEndpoint.SystemName` and avoid string parsing. I also opted to move the single mapping method into same class instead.

Reply:
Funny I hadn't noticed before that main console host doesn't use the same pattern as service endpoints. The console for China does follow the service endpoint. I'm going to guess some legacy issue there. 

I added a new comment on how you can validate the region for the stack is valid for the `https://console.aws.amazon.com/` console endpoint.